pkgs/by-name/c2/c2000-cgt/package.nix

```
@@ -0,0 +1,55 @@
+{
+  buildFHSEnv,
+  lib,
+  stdenvNoCC,
+}:
+
+let
+  fhs = buildFHSEnv {
+    name = "fhs-shell";
+  };
+
+  package = stdenvNoCC.mkDerivation rec {
+    pname = "cgt_c2000";
+    version = "22.6.1.LTS";
+
+    filename = "ti_${pname}_${version}_linux-x64_installer.bin";
+
+    src = builtins.fetchurl {
+      url = "https://dr-download.ti.com/software-development/ide-configuration-compiler-or-debugger/MD-xqxJ05PLfM/${version}/${filename}";
+      sha256 = "sha256:1nikmcgja2i22qk4alclfqiaqxwc7k0m9hwl5pviiv43bivb3rw7";
```

Use SRI hashes (convert via `nix hash convert --hash-algo sha256`)
```suggestion
      hash = "sha256-h+exdlyD7Bj3LZTDVME8jHesInaUUUUmFiIKJR+rM9o=";
```